### ISSUE TITLE
--no-cache option not working

### ISSUE THREAD

**Issue by Tonio-lavanda** [apancutt/deploy-aws-s3-cloudfront#9]
Hello,

Thanks for this package, it is very useful and works well.

I was playing with the `--no-cache` option and could not get it working. Whatever the value I set for this arg, it is always an empty array. After a lot of experimentation, I notice the following:

 - any option starting with `no-` is ignore by `yargs.options` (I can't find out why)

I tried to replace in your code `no-cache` by `no-whatever` and could never see this argument being set.

If I replace `no-cache` by `noCache`, this is working as expected.

I'm tried with 2 different version of node: v10.11.0 & v12.5.0

Maybe it is due to something like this: https://yargs.js.org/docs/#api-booleankey (there seem to be a --no- prefix for boolean, I can't see anything for array but it might be linked).

Anyway I think an easy fix for this is to replace in `deploy-aws-s3-cloudfront.js` at line 52:

Old:
```
  .option('no-cache', {
    type: 'array',
    describe: 'Disable caching of specified S3 path(s)',
    default: [],
  })
```

New:
```
  .option('noCache', {
    type: 'array',
    describe: 'Disable caching of specified S3 path(s)',
    default: [],
  })
```

This worked for me.

**Comment by Tonio-lavanda** [comment]
I submitted a PR for this: https://github.com/apancutt/deploy-aws-s3-cloudfront/pull/11

**Comment by apancutt** [comment]
Fixed in 2.2.2. The `--no-cache` option has been renamed to `--cache-control-no-cache`.
